doc/VSSDK004.md

```diff
@@ -0,0 +1,43 @@
+# VSSDK004 Use BackgroundLoad flag in ProvideAutoLoad attribute for asynchronous auto load
+
+In order to provide performance guarentees in critical scenarios such as startup and solution load,
+Visual Studio will deprecate synchronous auto load requests in a future version.
+
+This analyzer flags all use of ProvideAutoLoad attributes that doesn't provide BackgroundLoad flag or
```

Comment:
grammar: "that doesn't provide *a* BackgroundLoad flag..."

Also, I encourage you to use backticks to quote all type/member names.